cli/README.md

```diff
@@ -9,10 +9,21 @@ and [here](https://github.com/googleinterns/spec-math/tree/master/library).
 
 ### For Unix Based Operating Systems
 
-First, navigate to the [cli folder](.) in your terminal. Then run the following commands:
+As the Spec Math Library has not yet been uploaded to Maven, you will need to first need to navigate
+to the [library folder](../library) in your terminal. Then run the following commands:
 
 ```
-mvn clean install compile
+mvn clean
```

Comment:
Why not just run mvn clean install (that should run compile too)

Reply:
good point! Was just about to add this change